api/proto/teleport/okta/v1/okta_service.proto

```diff
@@ -120,16 +120,20 @@ message CreateIntegrationRequest {
   string scim_token = 3;
   // Enable synchronization of access lists.
   bool enable_access_list_sync = 4;
-  // Enable user synchronization.
+  // Enable user synchronization from Okta to Teleport.
   bool enable_user_sync = 5;
-  // Enable synchronization of app groups.
+  // Enable synchronization of app groups from Okta to Teleport.
   bool enable_app_group_sync = 6;
   // Settings related to access list synchronization.
   AccessListSettings access_list_settings = 7;
   // Option to reuse an existing connector.
   string reuse_connector = 8;
   // SSO metadata URL for integration.
   string sso_metadata_url = 9;
+  // Enable sync from Teleport to Okta. If disabled it will prevent creating Okta assignments which
+  // in turn will disable JIT Access Requests. Makes sense only when apps and groups sync is
+  // enabled.
+  bool enable_bidirectional_sync = 10;
```

Comment:
Why is the plugin setting enabled by default, while this is disabled by default?

Same for the field below.

Reply:
We need that for backward compatibility.

Reply:
The other way around because of backwards compatibility too? Could you elaborate a bit more why they need to be backward-compatible but in different ways from each other?

Reply:
plugin is a resource stored in the backend. So if we read a plugin with disable_bidirectional_sync, it will mean bidirectional sync is on (current behaviour). When it comes to the requests ({Create,Update}IntegrationRequest) we have a bit more freedom because they are being used only from the UI. We don't use them in `tctl` yet. Same situation as with {enable,disable}_app_group_sync.

I share your concerns, but I prefer consistency over "best practices" here.

Reply:
I can't say I follow the reasoning or what best practices you refer to, but I appreciate the explanations nonetheless.

Reply:
I'd say we should prefer enable flags rather than disable as this is less confusing. That's what I meant by "best practice".